src/oc_erchef/apps/oc_chef_wm/src/oc_chef_wm_acl_permission.erl

```diff
@@ -111,15 +111,16 @@ from_json(Req, #base_state{organization_guid = OrgId,
     case update_from_json(AclState, Part, OrgId) of
         forbidden ->
             {{halt, 400}, Req, State};
-        bad_actor ->
-            % We don't actually know which one, so can't use a more specific
-            % message without checking every single object (which we don't yet
-            % do efficiently); we only get back a list of Ids that has a
-            % different length than the list of names we started with.
-            Msg = <<"Invalid/missing actor in request body">>,
-            Msg1 = {[{<<"error">>, [Msg]}]},
-            Req1 = wrq:set_resp_body(chef_json:encode(Msg1), Req),
-            {{halt, 400}, Req1, State#base_state{log_msg = bad_actor}};
+        {ambiguous_actor, Actors} ->
+            Msg = {[{<<"error">>, iolist_to_binary([<<"The actor(s)">>, Actors,
```

Comment:
Yep, this is one of the times when a TODO would have helped me remember to come back :D Fix in flight.